packages/story-editor/src/components/localAutoSaveHandler/localAutoSave.js

```diff
@@ -97,6 +97,18 @@ function LocalAutoSave() {
     setBackup(null);
   };
 
+  const hasNewChangesTracker = useRef(false);
```

Comment:
`hadNewChanges`?

Reply:
Sure, why not. Change in [2cadd17](https://github.com/GoogleForCreators/web-stories-wp/pull/12417/commits/2cadd173361b76169c305013846e6564bf0bb534)